2018/proposal.md

```diff
@@ -0,0 +1,90 @@
+developers helps developers provide better support for beginners in Python and its libraries, as well as
+giving educators the rare opportunity to develop their own technical skills further.
+
+As in previous years, talk topics at the miniconf will be open to any educational use of Python but the core
+focus will continue to be on primary and secondary school education. This includes both formal classroom
+instruction and extracurricular activities.
+
+However, some changes and additions are also being proposed based on our experiences in previous iterations
+of the event.
+
+Event Organisers
+----------------
+
+* Lead organiser: Nick Coghlan
+  * 2015 Education Seminar founder & coordinator, 2016/17 co-coordinator with Katie Bell
+  * CPython core developer
+  * Former member of the Python Software Foundation Board of Directors (2014-16)
+
+* Co-organiser: Nicky Ringland? (Bruce Fuda?, Amanda Hogan?)
+  * Australian Computing Academy: https://aca.edu.au/about.html
+
+Event Outline
+-------------
+
+Prior to the afternoon tea break, the 2018 Education Seminar will follow the same
+structure as in previous years: an invited speaker prior to morning tea, 3 speakers
+from the Call for Proposals prior to lunch, then 3 more speakers prior to afternoon
+tea.
+
+For the final session, we'd like to work with the main conference organisers to
+arrange to run a "Student Showcase", inspired by the well-received student
+presentations at PyCon Australia 2017.
+
+The general idea of this part of the proposal would be to:
+
+* Provide a way for teachers & senior secondary students to nominate as potential
+  presenters at the Showcase (similar to the way Gungahlin College students did
+  through the regular CfP for 2017)
+* Accept up to 6 projects for presentation (these would each be given a 15 minute
+  presentation slot *without* any question & answer time, and a ~5 minute break
+  in between slots. Moving back and forth between the Student Showcase and the other
+  specialist tracks would generally be discouraged)
+* Work with the conference organisers to provide a way for teachers, family members,
+  and students to attend just the showcase session for a relatively nominal cost
+  (e.g. $15)
+
+Note that while in 2017 there was some question as to whether PyCon Australia
+volunteer organisers needed to apply for and receive the appropriate state credentials
+for working with children due to some speakers and volunteers being under 18, adding
+the Student Showcase to the program would make this requirement unambiguous:
+since the Student Showcase would be intended specifically for high school students,
+the Education Seminar organisers and volunteers and the main conference organisers
+would definitely need those approvals in place. Organisers of other specialist tracks
+may not need them, but it would likely be simpler to make it a general requirement
+for all volunteers at the event.
```

Comment:
The volunteer Working with Children Check (or other state equivalent) is free, too, which is nice.

Technically, we wouldn't need to [require WWCC](https://www.kidsguardian.nsw.gov.au/child-safe-organisations/working-with-children-check/about-the-check/exemptions) for those coming from out of state, but I would strongly recommend it anyway. 

> Interstate visitors:
> can work or volunteer at a one-off event such as a jamboree, sporting or religious event or tour, for up to 30 days a year without a NSW Working With Children Check
> can work or volunteer in any child-related work for up to 30 days a year, if the person holds an interstate Working With Children Check, or is exempt from the requirement to have such a check in his or her home jurisdiction

Reply:
The other states have similar requirements, so we still need the equivalent in our home state though. For example, https://www.bluecard.qld.gov.au/volunteers/Churchesclubsandassociationsinvolvingchildren.html seems to be the most applicable category here in Qld.

We determined in 2017 that we *didn't* need them because we weren't "conducting activities that mainly involve children". I believe that will change, at least for the Education Seminar, as soon as we introduce the Student Showcase. I don't think that's a problem - we just need to be aware of it far enough in advance that folks have plenty of time to apply for one, and to ensure we have the right information from Linux Australia to include on the form.